nalanet/nalanet/organizations/GA/gateway/connection-org1.yaml

```diff
@@ -0,0 +1,63 @@
+---
+name: test-network-org1
+version: 1.0.0
+client:
+  organization: Org1
+  connection:
+    timeout:
+      peer:
+        endorser: '300'
+organizations:
+  Org1:
+    mspid: Org1MSP
+    peers:
+    - peer0.org1.example.com
+    certificateAuthorities:
+    - ca.org1.example.com
+peers:
+  peer0.org1.example.com:
+    url: grpcs://localhost:7051
+    tlsCACerts:
+      pem: |
+          -----BEGIN CERTIFICATE-----
+          MIICJjCCAc2gAwIBAgIUVukFb2NalhHjp4cx8T47EyPRj10wCgYIKoZIzj0EAwIw
+          cDELMAkGA1UEBhMCVVMxFzAVBgNVBAgTDk5vcnRoIENhcm9saW5hMQ8wDQYDVQQH
+          EwZEdXJoYW0xGTAXBgNVBAoTEG9yZzEuZXhhbXBsZS5jb20xHDAaBgNVBAMTE2Nh
+          Lm9yZzEuZXhhbXBsZS5jb20wHhcNMjEwNDAxMTA1MzAwWhcNMzYwMzI4MTA1MzAw
+          WjBwMQswCQYDVQQGEwJVUzEXMBUGA1UECBMOTm9ydGggQ2Fyb2xpbmExDzANBgNV
+          BAcTBkR1cmhhbTEZMBcGA1UEChMQb3JnMS5leGFtcGxlLmNvbTEcMBoGA1UEAxMT
+          Y2Eub3JnMS5leGFtcGxlLmNvbTBZMBMGByqGSM49AgEGCCqGSM49AwEHA0IABF86
+          cUpFpQTWImevPjSO+OXypA0Y0BiWtxiFynYH/Ap8lutL0qnzrlHU62bYIRN8oI9w
+          DIpWpovFve02PF7lnmmjRTBDMA4GA1UdDwEB/wQEAwIBBjASBgNVHRMBAf8ECDAG
+          AQH/AgEBMB0GA1UdDgQWBBTAwLBJICVMYlm4NNYHJWoxGZmKFjAKBggqhkjOPQQD
+          AgNHADBEAiAB/SMG6j8GO7NcC4R4qVBE9VtRcC5qLjWSMum9i2LvCAIgdqTVKElQ
+          AJRUwvdDj/ZnNMQS+uGVchta864R4cEGwCk=
+          -----END CERTIFICATE-----
+          
```

Comment:
Will we have to remove/hide these once we deploy?

Reply:
yea that's just a dummy thing I was trying.  i thought I had already removed that. thanks for pointing it out. i will go ahead and remove it now.

Reply:
wait, is there a way i could make another PR off of this?

Reply:
Actually, let's not merge this PR now. it needs a lot more work. i only did half of the entire thing.